### PR TITLE
improve: added the support for trailer headers

### PR DESCRIPTION
**Pull request by tokers**

### CODE REVIEW

**Comment by tokers** [comment]
https://github.com/tokers/lua-resty-requests/issues/28

**Comment by tokers** [comment]
@membphis

**Comment by membphis** [comment]
I will make a try later, many thx.

Can we add a test case for this change?

**Comment by tokers** [comment]
Sure, I will add some test cases to cover it later.

File: t/03-frame.t [hunk 1]
```diff
@@ -31,6 +31,16 @@ our $http_config = << 'EOC';
                 ngx.header["Cookie"] = ngx.var.http_cookie
             }
         }
+
+        locatin = /t3 {
```
Comment (membphis) [inline t/03-frame.t]: typo: location

Comment (tokers) [reply t/03-frame.t]: fixed.

**Review by membphis — APPROVED** [review]

**Comment by tokers** [comment]
Merged.

**Review by lsc153 — COMMENTED** [review]
http2.lua  line 419   if not session.done  should be  stream.done

**Pull request merged**